### PR TITLE
Feat:-  cover image and curr time

### PR DESCRIPTION
**Pull request by shubhamkumar648**
- add cover image changes with help of unsplash API 
- add digital clock

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@import url("https://grow-n-know.netlify.app/styles/main.css");
```
Comment (kjais1720) [inline src/App.css]: import any external css file in the html for better performance.

File: src/utils/api.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+export const getImage = async (setImageData) => {
+  try {
+    const {
+      data: { urls, location, discrptions }
```
Comment (kjais1720) [inline src/utils/api.js:7]: is it a typo?

File: src/utils/dist/api.dev.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  value: true
+});
+exports.getImage = void 0;
+
+var _axios = _interopRequireDefault(require("axios"));
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }
+
+var getImage = function getImage(setImageData) {
+  var _ref, _ref$data, urls, location, discrptions;
```
Comment (kjais1720) [inline src/utils/dist/api.dev.js]: same typo here

**Review by kjais1720 — CHANGES_REQUESTED** [review]
move the `.env` file into `.gitignore`

File: src/App.css (hunk 1 above)
Comment (shubhamkumar648) [reply src/App.css]: will do that thanks for suggestion

File: src/utils/api.js (hunk 2 above)
Comment (shubhamkumar648) [reply src/utils/api.js:7]: No ,here i did destructure accordingly which is  name given in data (api )

File: src/utils/dist/api.dev.js (hunk 3 above)
Comment (shubhamkumar648) [reply src/utils/dist/api.dev.js]: this file is not valid , forgot to delete

**Review by kjais1720 — APPROVED** [review]
Looks good now 👍

**Review by RahulSingh9131 — COMMENTED** [review]
No changes required, just remove the commented code.

File: src/App.css [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+
+
+/* .cover-img {
+    height: 100vh;
+    position: fixed;
+    width: 100vw;
+    z-index: -1;
+} */
```
Comment (vaibhav-mahalle) [inline src/App.css]: remove unnecessary code while raising pr.

File: src/routes/dist/Routes.dev.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+// import React from 'react'
```
Comment (vaibhav-mahalle) [inline src/routes/dist/Routes.dev.js:1]: remove commented code while raising pr.

**Review by vaibhav-mahalle — COMMENTED** [review]
added minor suggestions, please look into them.

**Pull request merged**